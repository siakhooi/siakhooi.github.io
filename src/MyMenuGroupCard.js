import React from 'react'
import PropTypes from 'prop-types'

MyMenuGroupCard.propTypes = {
  children: PropTypes.array
}
function MyMenuGroupCard (props) {
  return (
    <div className="MyMenuGroupCard">{props.children}</div>
  )
}

export default MyMenuGroupCard
