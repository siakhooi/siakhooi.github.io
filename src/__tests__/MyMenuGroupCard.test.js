import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuGroupCard from '../MyMenuGroupCard'

it('MyMenuGroupCard/1', () => {
  const component =
    renderer.create(
      <MyMenuGroupCard><div>ABC</div><div>ABC</div></MyMenuGroupCard>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
