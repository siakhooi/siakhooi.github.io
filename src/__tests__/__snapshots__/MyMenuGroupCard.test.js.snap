// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuGroupCard/1 1`] = `
<div
  className="MyMenuGroupCard"
>
  <div>
    ABC
  </div>
  <div>
    ABC
  </div>
</div>
`;
